build.py

```diff
@@ -192,6 +192,9 @@ def main():
     # Enter source tree to run build commands
     os.chdir(source_tree)
 
+    # Set "DEPOT_TOOLS_WIN_TOOLCHAIN" variable so gn gen doesn't return errors.
+    os.environ['DEPOT_TOOLS_WIN_TOOLCHAIN'] = "0"
```

Comment:
Can we set this inside `_run_build_process()` instead? This is where we are already setting environment variables for build processes. Also, instead of setting the current process's environment variables (via `os.environ`), I'd like to have the variable exported from the temporary batch script, i.e. add a batch command to the `cmd_input` list of `_run_build_process()` that will set the environment variable. This will make the code a little cleaner.

Reply:
Done.